src/onedb/local/4.90.0_to_5.3.80.rb

```diff
@@ -15,17 +15,20 @@
 #--------------------------------------------------------------------------- #
 
 
+
 require 'set'
 require 'base64'
 require 'zlib'
 require 'pathname'
 
+require 'yaml'
 require 'opennebula'
 
 $: << File.dirname(__FILE__)
 
 include OpenNebula
 
+EC2_DRIVER_CONF = "#{ETC_LOCATION}/ec2_driver.conf"
```

Comment:
Move to a normal variable inside `feature_5136` method as it is not used anywhere else